docs/architecture docs/subsystems/7. Migration System.md

```
@@ -4,69 +4,79 @@
-  "ops": [
-    {
-      "op": "createTable",
-      "table": "user",
-      "pre": [{ "check": "tableNotExists", "table": "user" }],
-      "post": [{ "check": "tableExists", "table": "user" }]
-    }
-  ]
-}
+The migration directory on disk:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)